Neos.Media.Browser/Classes/ViewHelpers/ThumbnailViewHelper.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Neos\Media\Browser\ViewHelpers;
+
+/*
+* This file is part of the Neos.Media.Browser package.
+*
+* (c) Contributors of the Neos Project - www.neos.io
+*
+* This package is Open Source Software. For the full copyright and license
+* information, please view the LICENSE file which was distributed with this
+* source code.
+*/
+
+use Neos\Media\Domain\Model\AssetSource\AssetProxy\AssetProxy;
+use Neos\FluidAdaptor\Core\ViewHelper\AbstractTagBasedViewHelper;
+
+/**
+ * Renders an <img> HTML tag from a given asset proxy instance
+ *
+ * = Examples =
+ *
+ * <code title="Rendering an asset proxy thumbnail">
+ * <mediaBrowser:thumbnail assetProxy="{assetProxyObject}" alt="a sample asset" />
+ * </code>
+ * <output>
+ * <img src="https://my-asset-management.com/thumbnails/espresso.jpg" width="120" height="180" alt="a sample asset without scaling" />
+ * </output>
+ */
+class ThumbnailViewHelper extends AbstractTagBasedViewHelper
```

Comment:
Doesn't this exist in the Media package?

Reply:
Yes, will but this one is different. I'll add a comment to declare it as "internal" as I don't want to add this functionality too early into the official / public thumbnail view helper.